File: Easy/1431.Greatest_Number_of_candies.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        x=max(candies)
        arr=[]
        for i in range(len(candies)):
            if (candies[i]+extraCandies>=x):
                arr.append(True)
            else :
                arr.append(False)
        return arr

        """
        :type candies: List[int]
        :type extraCandies: int
        :rtype: List[bool]
        """
        

File: Easy/1431.Greatest_number_of_candies.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
bool* kidsWithCandies(int* candies, int candiesSize, int extraCandies, int* returnSize) {
    *returnSize=candiesSize;
    bool *arr=(bool*) malloc ((*returnSize)*sizeof(bool));
    int i,j,max=candies[0];
    for (i=0;i<candiesSize;i++){
        if(max<candies[i]){
            max=candies[i];
        }
    }
    for (i=0;i<candiesSize;i++){
        if (candies[i]+extraCandies>=max){
            arr[i]=true ;
            
        }
        else {
            arr[i]=false;
        }
    }
    return arr;
}